### PR TITLE
Fix Various Lint Cops 

### PR DESCRIPTION
**Pull request by archanaserver**
#### What are the changes introduced in this pull request?
This PR addresses several Lint offenses across the codebase, check commit for more details.

#### Considerations taken when implementing this change?
Fixed deprecated constants and other linting issues.

#### What are the testing steps for this pull request?
bundle exec rubocop

### CODE REVIEW

**Comment by ianballou** [comment]
Looks like there are some remaining Lint cops failing.

**Comment by archanaserver** [comment]
> Looks like there are some remaining Lint cops failing.

Fixed it!

**Comment by archanaserver** [comment]
> > Looks like there are some remaining Lint cops failing.
> 
> Fixed it!

hie @ianballou would you mind taking a look again?

File: app/lib/actions/katello/host/hypervisors_update.rb [hunk 1]
```diff
@@ -33,11 +33,10 @@ def run
...
 
-          @duplicate_uuid_hypervisors.each do |hypervisor, existing_host|
-            fail _("Host creation was skipped for %s because it shares a BIOS UUID with %s. " \
-                   "To report this hypervisor, override its dmi.system.uuid fact or set 'candlepin.use_system_uuid_for_matching' " \
-                   "to 'true' in the Candlepin configuration." % [hypervisor[:name], existing_host.name])
-          end
+          hypervisor, existing_host = @duplicate_uuid_hypervisors.first
+          fail _("Host creation was skipped for %s because it shares a BIOS UUID with %s. " \
+                 "To report this hypervisor, override its dmi.system.uuid fact or set 'candlepin.use_system_uuid_for_matching' " \
+                 "to 'true' in the Candlepin configuration." % [hypervisor[:name], existing_host.name])
```
Comment (ianballou) [inline app/lib/actions/katello/host/hypervisors_update.rb:39]: More investigation is needed but I think the changes here somehow caused `hypervisor` to become `nil`.

Comment (archanaserver) [reply app/lib/actions/katello/host/hypervisors_update.rb]: @ianballou i've made changes to handle this.

**Review by qcjames53 — APPROVED** [review]
Hi Archana, thanks for the changes. The new rule + the refactors all look good to me :)

I'm going to ACK and squash merge.

**Comment by archanaserver** [comment]
Thanks @ianballou and @qcjames53 for the review!

**Pull request merged**